src/test/regress/sql/multi_extension.sql

```diff
@@ -175,3 +175,131 @@ ALTER EXTENSION citus UPDATE;
 
 -- if cache is invalidated succesfull, this \d should work without any problem
 \d
+
+\c - - - :master_port
+
+CREATE VIEW table_fkeys AS
+SELECT r.conname AS "Constraint",
+       pg_catalog.pg_get_constraintdef(r.oid, true) AS "Definition",
+       conrelid AS "relid"
+FROM pg_catalog.pg_constraint r
+WHERE r.contype = 'f';
+
+-- create views used to describe relations
+CREATE VIEW table_attrs AS
+SELECT a.attname AS "name",
+    pg_catalog.format_type(a.atttypid, a.atttypmod) AS "type",
+    (SELECT substring(pg_catalog.pg_get_expr(d.adbin, d.adrelid) for 128)
+     FROM pg_catalog.pg_attrdef d
+     WHERE d.adrelid = a.attrelid AND d.adnum = a.attnum AND a.atthasdef) AS "default",
+    a.attnotnull AS "notnull",
+    a.attrelid AS "relid"
+FROM pg_catalog.pg_attribute a
+WHERE a.attnum > 0 AND NOT a.attisdropped
+ORDER BY a.attnum;
```

Comment:
It'd be better to refer to information_schema.columns here.  Fine with having table_attrs as a view over that, but that way we won't have to update it several times over the next releases.